scripts/camera.lua

```diff
@@ -150,6 +166,8 @@ function Camera.followTracker(playerSettings, player, camera, tracker, disableSm
         camera.centerPos = tracker.centerPos
         camera.zoom = Camera.zoom(camera, tracker)
     end
+
+    Camera.refreshChartTags(player, camera, captureBox, camera.centerPos, camera.zoom)
```

Comment:
We could do a check to see if old position and zoom match new/current position and zoom. If so, no need to replace the tags (I have no idea how 'expensive' the destroy/create tag operations are...)

Reply:
Tags seem functionally free, we do this update very rarely, and it is swamped by the cost of the screenshot itself. I doubt the extra  code lines would be worth the wins.

Reply:
true, compared to the screenshot it will be neglectable